From 41ec2300e499c731f8576d49c67da4a5f43e64ef Mon Sep 17 00:00:00 2001
From: Orlando Vazquez <ovazquez@joyent.com>
Date: Thu, 28 Mar 2019 10:19:24 -0700
Subject: [PATCH] TRITON-1289 sdc-vmapi runtests needs a mechanism to run
 without fabrics Reviewed by: Todd Whiteman <todd.whiteman@joyent.com>
 Approved by: Todd Whiteman <todd.whiteman@joyent.com>

---
 package.json          | 2 +-
 test/vms.full.test.js | 7 +++----
 2 files changed, 4 insertions(+), 5 deletions(-)

diff --git a/package.json b/package.json
index ccfa1ae..95cfd35 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "vmapi",
   "description": "VMs API",
-  "version": "9.7.2",
+  "version": "9.7.3",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
diff --git a/test/vms.full.test.js b/test/vms.full.test.js
index f1712dd..c926562 100644
--- a/test/vms.full.test.js
+++ b/test/vms.full.test.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2018, Joyent, Inc.
+ * Copyright (c) 2019, Joyent, Inc.
  */
 
 // var test = require('tap').test;
@@ -2882,14 +2882,13 @@ exports.destroy_test_vms_final = function (t) {
 exports.find_fabric_network = function (t) {
     assert.arrayOfObject(NETWORKS, 'NETWORKS');
 
-    // Find a fabric network that is owned by CUSTOMER.
+    // Find a fabric network that is owned by CUSTOMER. If none is found, skip
+    // the tests that depend on the availability of one.
     fabricNetwork = NETWORKS.find(function _findFabricNetwork(n) {
         return n.fabric === true && Array.isArray(n.owner_uuids) &&
             n.owner_uuids.indexOf(CUSTOMER) >= 0;
     });
 
-    t.ok(fabricNetwork, 'found a fabric network');
-
     if (fabricNetwork) {
         assert.object(fabricNetwork, 'fabricNetwork');
         assert.string(fabricNetwork.name, 'fabricNetwork.name');
-- 
2.21.0

